docs/cli.txt

```diff
@@ -15,6 +15,9 @@ The ``crash`` binary supports several command line arguments.
 +---------------------------------------+--------------------------------------------------+
 | ``-U``, ``--username``                | The username to authenticate in the database     |
 +---------------------------------------+--------------------------------------------------+
+| ``-W``, ``--password``                | Force prompt for password. Otherwise it prompts  |
+|                                       | for password automatically.                      |
```

Comment:
in case the hba configuration of the server requires "password authentication" for the specified user ?